airqo-platform/AirQo-api
# reduced prediction points

**_WHAT DOES THIS PR DO?_**

Reduce the number of air quality predictions per airqloud to less than 100

**_HOW DO I TEST OUT THIS PR?_**

- create a virtual environment
- install all requirements, src/gp-model/requirements.txt
- get the .env and test using development environment (**note that: for the event API we now use .../api/v2/..**)
- run the src/gp-model/main.py

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

The [**heatmap** API](https://staging-platform.airqo.net/api/v1/predict/heatmap)
